spartan/aztec-chaos-scenarios/templates/network-shaping.yaml

```diff
@@ -57,7 +57,7 @@ metadata:
   name: {{ .Values.global.targetNamespace }}-packet-loss
   namespace: {{ .Values.global.chaosMeshNamespace }}
   labels:
-    {{- include "network-shaping.labels" . | nindent 4 }}
+    {{- include "aztec-chaos-scenarios.labels" . | nindent 4 }}
   annotations:
     "helm.sh/resource-policy": keep
```

Comment:
I understand this annotation to mean that this resource is left behind when he helm chart is uninstalled. Is this intentional? 

Would applying `"helm.sh/hook": post-install,post-upgrade` and `"helm.sh/hook-weight": "<some-weight>"` be sufficient for the scenarios?